rust/src/plutus.rs

```diff
@@ -1211,4 +1270,12 @@ mod tests {
         witness_set.set_plutus_data(&list);
```

Comment:
@kodemill why was the transaction body added in this test? Was the mismatch causing an issue anywhere or was it just added in conservatively? From what I know the transaction body's cbor is canonicized before hashing to compute the id, and the plutus datums are not canonicized so we must not change their representation like we were. Your PR #228 made us in line with the CLI for lists but this turned out to be more of a band-aid.

Reply:
Yes, the witness set change was just a conservative addition, to be in line with the CLI, feel free to revert it!